ini/stm32f1.ini

```diff
@@ -332,17 +332,15 @@ upload_protocol = serial
 platform                    = ${common_stm32.platform}
 extends                     = stm32_variant
 board                       = genericSTM32F103VE
-board_build.variant         = MARLIN_F103Vx
+board_build.variant         = MARLIN_F103VE_LONGER
 board_build.rename          = project.bin
 board_build.offset          = 0x10000
 board_upload.offset_address = 0x08010000
-build_flags                 = ${stm32_variant.build_flags}
-                              -DMCU_STM32F103VE -DU20 -DTS_V12 -DLED_BUILTIN=PC2 -UPIN_WIRE_SDA
-                              -UPIN_WIRE_SCL -DPIN_WIRE_SDA=PB11 -DPIN_WIRE_SCL=PB10
-                              -DHAL_DAC_MODULE_DISABLED -DHAL_I2S_MODULE_DISABLED
-build_unflags               = ${stm32_variant.build_unflags}
-                              -DUSBCON -DUSBD_USE_CDC -DHAL_PCD_MODULE_ENABLED
+build_flags                 = ${stm32_variant.build_flags} -DMCU_STM32F103VE -DU20 -DTS_V12
+build_unflags               = ${stm32_variant.build_unflags} -DUSBCON -DUSBD_USE_CDC -DHAL_PCD_MODULE_ENABLED
 extra_scripts               = ${stm32_variant.extra_scripts}
+monitor_speed               = 250000
+debug_tool                  = stlink
```

Comment:
we have a stlink connector.. its why i set it as default...

Reply:
![image](https://user-images.githubusercontent.com/145119/127755622-8fc8f1c9-c088-4b4a-b9bb-1e58f8fcdf35.png)